### PR TITLE
[TACHYON-679] Move shared utilities for tests to other packages to get rid of test-jar

### PR DESCRIPTION
**Pull request by uronce-cc**
https://tachyon.atlassian.net/browse/TACHYON-679

### CODE REVIEW

**Comment by calvinjia** [comment]
@uronce-cc Thanks for the patch, it seems like some jars are missing from integration tests?

**Comment by apc999** [comment]
```
[ERROR] /home/jenkins/workspace/Tachyon-Pull-Request-Builder/integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java:[30,15] cannot find symbol
[ERROR] symbol:   class TestUtils
[ERROR] location: package tachyon
[ERROR] /home/jenkins/workspace/Tachyon-Pull-Request-Builder/integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java:[76,30] cannot find symbol
[ERROR] symbol:   variable TestUtils
[ERROR] location: class tachyon.worker.block.meta.TieredStoreIntegrationTest
[ERROR] /home/jenkins/workspace/Tachyon-Pull-Request-Builder/integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java:[96,30] cannot find symbol
[ERROR] symbol:   variable TestUtils
[ERROR] location: class tachyon.worker.block.meta.TieredStoreIntegrationTest
[ERROR] /home/jenkins/workspace/Tachyon-Pull-Request-Builder/integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java:[103,30] cannot find symbol
[ERROR] symbol:   variable TestUtils
[ERROR] location: class tachyon.worker.block.meta.TieredStoreIntegrationTest
[ERROR] /home/jenkins/workspace/Tachyon-Pull-Request-Builder/integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java:[114,30] cannot find symbol
[ERROR] symbol:   variable TestUtils
[ERROR] location: class tachyon.worker.block.meta.TieredStoreIntegrationTest```
Should be an easy fix

**Comment by uronce-cc** [comment]
`TestUtils` is brought back after merging master, removed.

**Comment by apc999** [comment]
+1 LGTM.

Thanks a lot for the patch! It really makes POM files cleaner.

File: common/src/main/java/tachyon/util/CommonUtils.java [hunk 1]
```diff
@@ -79,4 +82,9 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
     Constructor<T> ctor = cls.getConstructor(ctorClassArgs);
     return ctor.newInstance(ctorArgs);
   }
+
+  public static int getToMasterHeartBeatIntervalMs(TachyonConf tachyonConf) {
+    return tachyonConf
+        .getInt(Constants.WORKER_TO_MASTER_HEARTBEAT_INTERVAL_MS, Constants.SECOND_MS);
+  }
```
Comment (calvinjia) [inline common/src/main/java/tachyon/util/CommonUtils.java]: This doesn't seem too much of a utility method (aside from reducing the line length), perhaps add a TODO to remove this and just use the actual `getInt`

File: common/src/main/java/tachyon/util/io/BufferUtils.java [hunk 2]
```diff
@@ -69,4 +71,78 @@ public static ByteBuffer generateNewByteBufferFromThriftRPCResults(ByteBuffer da
   public static void putIntByteBuffer(ByteBuffer buf, int b) {
     buf.put((byte) (b & 0xFF));
   }
+
+  public static byte[] getIncreasingByteArray(int len) {
```
Comment (calvinjia) [inline common/src/main/java/tachyon/util/io/BufferUtils.java]: We should document all these utilities, I've opened a ticket: https://tachyon.atlassian.net/browse/TACHYON-697

**Comment by calvinjia** [comment]
+1 LGTM, thanks for addressing those long standing TODOs!

**Pull request merged**